src/pages/pt-BR/reference/integrations-reference.md

```diff
@@ -0,0 +1,283 @@
+import bananaCSS from '@vitejs/official-banana-css-plugin';
+
+export default {
+  name: 'banana-css-integration',
+  hooks: {
+    'astro:config:setup': ({ updateConfig }) => {
+      updateConfig({
+        vite: {
+          plugins: [bananaCSS()],
+        }
+      })
+    }
+  }
+}
+```
+
+#### Opção `addRenderer`
+
+**Tipo:** `(renderer:` [`AstroRenderer`](https://github.com/withastro/astro/blob/fdd607c5755034edf262e7b275732519328a33b2/packages/astro/src/%40types/astro.ts#L872-L883) `) => void;`
+**Exemplos:** [`lit`](https://github.com/withastro/astro/blob/main/packages/integrations/lit/src/index.ts), [`svelte`](https://github.com/withastro/astro/blob/main/packages/integrations/svelte/src/index.ts), [`react`](https://github.com/withastro/astro/blob/main/packages/integrations/react/src/index.ts), [`preact`](https://github.com/withastro/astro/blob/main/packages/integrations/preact/src/index.ts), [`vue`](https://github.com/withastro/astro/blob/main/packages/integrations/vue/src/index.ts), [`solid`](https://github.com/withastro/astro/blob/main/packages/integrations/solid/src/index.ts)
+
+Uma função de callback para adicionar um renderizador de um framework de componentes (e.x. React, Vue, Svelte, etc). Você pode explorar os exemplos e definições de tipagem acima para opções mais avançadas, mas aqui estão as duas principais opções que você precisa estar ciente sobre:
+- `clientEntrypoint` - caminho para um arquivo que é executado no cliente sempre que seu componente é utilizado. Esta é principalmente utilizado para renderizar ou hidratar seu componente com JS.
+- `serverEntrypoint` - caminho para um arquivo que é executado durante requisições no lado do servidor ou builds estáticas sempre que seu componente é utilizado. Esta deve renderizar componentes para uma marcação estática, com hooks para hidratação aonde aplicável. [o callback `renderToString` do React](https://pt-br.reactjs.org/docs/react-dom-server.html#rendertostring) é um exemplo clássico.
+
+#### Opção `injectScript` 
+
+**Tipo:** `(stage: InjectedScriptStage, content: string) => void;`
+
+Uma função de callback para injetar uma string de conteúdo JavaScript em todas as páginas.
+
+O **`stage`** indica como este script (o `content`) deve ser inserido. Alguns stages permitem inserir scripts sem modificação, enquanto outros permitem otimizações durante [a etapa de bundling do Vite](https://vitejs.dev/guide/build.html):
+
+- `"head-inline"`: Injetado em uma tag script no `<head>` de cada página. **Não é** otimizado ou resolvido pelo Vite.
+- `"before-hydration"`: Importado no lado do cliente, antes do script de hidratação ser executado. Otimizado e resolvido pelo Vite.
+- `"page"`: Similar a `head-inline`, exceto que o script injetado é lidado pelo Vite e passa por bundle junto com quaisquer outras tags `<script>` definidas dentro de componentes Astro na página. O script será carregado com um `<script type="module">` no resultado final da página, otimizado e resolvido pelo Vite.
```

Comment:
Hi @victor0x16, I agree with your first suggestion but we are currently not translating `bundle` as well as `build` because of a discussion we had on our [Translation Guide document ](https://docs.google.com/document/d/1REr0Y2ZBOsxcVf6lSilTXgs1z3R4JgmyShaYvMW8qkQ/edit?disco=AAAAZUoPGhE) about translating words like `bundle` and `build` being unclear to a lot of people of the process made by Vite they refer to.

Reply:
```suggestion
- `"page"`: Similar a `head-inline`, exceto que o script injetado é transformado por Vite e passa por bundle junto com quaisquer outras tags `<script>` definidas dentro de componentes Astro na página. O script será carregado com um `<script type="module">` no resultado final da página, otimizado e resolvido pelo Vite.
```